### PR TITLE
DIRECTOR: Add detection entries for multiple demos and games

### PR DESCRIPTION
**Pull request by einstein95**

### CODE REVIEW

File: engines/director/resource.cpp [hunk 1]
```diff
@@ -435,6 +435,7 @@ void Window::loadStartMovieXLibs() {
 	if (strcmp(g_director->getGameId(), "warlock") == 0 && g_director->getPlatform() != Common::kPlatformWindows) {
 		g_lingo->openXLib("FPlayXObj", kXObj);
 	}
+	g_lingo->openXLib("FileIO", kXObj);
```
Comment (sev-) [inline engines/director/resource.cpp:438]: Why did you decide to put it by default?

File: engines/director/lingo/xlibs/miscx.cpp [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/*************************************
+ *
+ * USED IN:
+ * Pippi
```
Comment (sev-) [inline engines/director/lingo/xlibs/miscx.cpp:25]: This is a great addition. I think, we shall start adding it everywhere. Even an incomplete list would let us understand which titles to test.

**Review by sev- — APPROVED** [review]
Great work. Only one question regarding fileio.

**Comment by bluegr** [comment]
Nice work!
This should be split into multiple commits. There is a single big commit that includes not only detection, but functional related changes and additions.

Please, split the changes into more fine grained commits

**Review by bluegr — CHANGES_REQUESTED** [review]
Please split the changes in multiple commits

File: engines/director/resource.cpp (hunk 1 above)
Comment (einstein95) [reply engines/director/resource.cpp]: That was due to one Director movie that relied on being launched from the main projector that lacked any embedded movie. I realise that the functionality is there but I'll have to look at it again as I believe it wasn't working when I checked.

**Comment by einstein95** [comment]
@bluegr @sev- I've split the commits up, please double check I haven't broken anything

**Pull request merged**